generators/app/package-json.js

```diff
@@ -124,18 +124,16 @@ const packageJSON = (current, context) => {
       'ie 11'
     ],
     'scripts': _.assign({}, current.scripts, {
-      'prebuild': 'npm run clean',
```

Comment:
Previously we used to do this prebuild, but since we have multiple different types of "build" now, it doesn't make sense.